Bike.java

```diff
@@ -0,0 +1,26 @@
+package com.k7it;
+
+public class Bike {
+
```

Comment:
missing parameter constructor and attributes should be private access level. use the setters and getters for each attribute.